lib/angular/push-ng.js

```diff
@@ -13,7 +13,9 @@ angular.module('wfm.push.service', [])
   };
 
   var registerSuccess = function() {
-    //
+    if (1 === 1) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Correct one of the identical sub-expressions on both sides of operator "===" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS1764)